src/WebJobs.Script/Workers/ProcessManagement/WorkerProcess.cs

```diff
@@ -190,7 +190,7 @@ internal void BuildAndLogConsoleLog(string msg, LogLevel level)
             };
             if (WorkerProcessUtilities.IsConsoleLog(msg))
             {
-                _workerProcessLogger?.LogDebug(WorkerProcessUtilities.RemoveLogPrefix(msg));
+                _workerProcessLogger?.Log(level, WorkerProcessUtilities.RemoveLogPrefix(msg));
```

Comment:
This was an intentional change - as we should consider all Worker logs as System Logs - system logs are noisy for Cx but Debug is the default logLevel for Kusto - product team has access to all debug level system logs.

Reply:
Aren't those two separate things? I agree they are system logs, but normally I see us choose a level _and_ a category. The level can be "Error", "Warning", "Information", "Debug", etc. and the category can be either "System" or "User". In this case I thought `_workerProcessLogger` was used for system logs and `_consoleLogSource` (in the "else" case below) was used for user logs.

Without proper categories, it makes supportability more difficult for the workers because any warning/error the worker logs is ignored - users _can't_ see it and the support team _won't_ see it because they often filter by log level.

Finally - at least when I'm running locally the default log level for the "Worker" category is "warning", so it's not noisy at all in my experience.

Reply:
Interesting. @pragnagopa it does look like the change has an impact on the logs surfaced to the end user, which is something we're trying to address. Is it possible the original change had a broader impact than anticipated and left us with a gap that makes troubleshooting more difficult for customers?

Reply:
Looking at the previous and the test added, the intention was to force language worker logs to be treated as  system logs similar to what we do in the host.  Also, condition here ` if (WorkerProcessUtilities.IsConsoleLog(msg))` - is specifically to ensure the log as known prefix.

If you would like to go ahead with this change - I recommend scoping the change to language workers that the team owns - as of now they are shipped and packaged with the host - to avoid logging info level logs from language workers not owned by the team.

Reply:
That sounds good to me, I would like to go ahead with the change if possible - @ejizba is it possible to scope this as Pragna suggested?

Reply:
@ejizba - I do not think host has a list of language workers that are well known - open a separate for the recommendation above. Approving this PR!

Reply:
Filed this issue to follow up: https://github.com/Azure/azure-functions-host/issues/8109